src/Kafka/CreateKafkaInstance/components/InstanceInfo.tsx

```diff
@@ -53,126 +59,163 @@ export const InstanceInfo: VoidFunctionComponent<
   connectionRate,
   messageSize,
   onClickQuickStart,
+  streamingUnits,
 }) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation("create-kafka-instance");
 
   return (
     <TextContent>
-      <Text component={TextVariants.h3}>
-        {t("create-kafka-instance:instance_information")}
-      </Text>
-      <TextList component={TextListVariants.dl}>
-        {!isLoading && (
-          <Grid sm={6} lg={12} hasGutter>
-            {isTrial && (
-              <GridItem>
-                <TextListItem component={TextListItemVariants.dt}>
-                  {t("kafka:duration")}
-                </TextListItem>
-                <TextListItem component={TextListItemVariants.dd}>
-                  {t("create-kafka-instance:duration_value", {
-                    value: trialDurationInHours,
-                  })}
-                </TextListItem>
-              </GridItem>
+      <Card>
```

Comment:
Can you add the `isFlat` prop to the cards. This removes the shadow and creates just a solid border.